src/components/Toast/README.md

```diff
@@ -0,0 +1,21 @@
+# `<Toast />`
+
+All Toast related dependencies are contained within this folder, with the exception of 3rd party libraries and a shared hook `useKeyPress`.
+
```

Comment:
💭 Some recent work has included a ["Props" table](https://github.com/cremalab/crema-components/blob/9d0ff9dfa48550c085e474ca211491598a9edc24/src/components/Tabs/README.md#api)—what are your thoughts? Is that too much?

Reply:
I feel like with how few props it has, it might be, but I'm happy to follow the trend.

Reply:
Something we should probably standardize and add to the hygiene templates if we want to go down that path.